music21/figuredBass/notation.py

```diff
@@ -30,6 +30,33 @@
                      (2,): (6, 4, 2),
                      }
 
+prefixes = ['+', '#', '++', '##']
+suffixes = ['\\']
+
+modifiersDictXmlToM21 = {'sharp': '#',
```

Comment:
indent as 

```
modifiersDictXmlToM21 = {
    'sharp' :'#',
    'flat': 'b',
    ...etc.
}
```

so that all keys begin on the same column and have four spaces showing they're inside a top-level dictionary. (same below)